client/src/app.css

```diff
@@ -0,0 +1,79 @@
+:root {
```

Comment:
is it necessary to have a .css file when the styling can be done through a svelte component?
If global styling was the goal, the global prefix can be used in the svelte component.

Reply:
I think generally you want want to have a root css file to handle most of the styling so that you dont have to add the styling to every component. We can remove it later on if needed

Reply:
sounds good!